src/com/codingforcookies/armorequip/ArmorListener.java

```diff
@@ -103,12 +104,15 @@ public final void inventoryClick(final InventoryClickEvent e){
 		}
 	}
 
-	@EventHandler
+    @EventHandler(
+            priority = EventPriority.MONITOR,
+            ignoreCancelled = true
+    )
```

Comment:
Can this be formatted to one line?

Reply:
Yeah absolutely, though I usually format it like that for readability. It can all be collapsed onto one line if desired.

Reply:
@Arnuh Tagging so this notifies, forgot to do so previously.

Reply:
Still multi line and I just realized its set to MONITOR. We modify the outcome of this which literally goes against the entire definition of the Monitor priority. I assume its best to set it to HIGHEST so we modify it last? Not entirely sure if that's the best choice but I think it is. @NecroneUnderscore 

Wont merge until both are changed.

Reply:
@Arnuh My bad, should be highest.